notebooks/227-whisper-subtitles-generation/227-whisper-subtitles-generation.ipynb

```diff
@@ -120,10 +120,10 @@
     "mel = torch.zeros((1, 80, 3000))\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #39.        Extract audio signal from a given video file, then convert it to float,
</pre><p>please remove unrelated changes from your submission</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Done.Reverted that commit.